job-roles/lead-engineer.md

```diff
@@ -0,0 +1,55 @@
+# Lead Engineer
+
+## Overview
+
+At MayaData we believe that the best way to deliver storage and related services to containerized and cloud-native environments is with containerized and cloud-native architectures.  Everything we do is focused on providing data agility and simplifying the daily operation of Kubernetes SRE’s and DevOps teams using Kubernetes as a data layer.  We enable our users to handle complex data management operations easily, and we have become open source leaders in building Kubernetes into a first-class data platform.  
+
+Lead engineers are the first level full time engineers at MayaData with exceptional knowledge on product and technologies used. This provides the change for individual to grow from team level to across multiple teams and company. Below are the expectations from them in-general irrespective of the kind of job role. They are also expected to fill the [sr software engineers](sr-software-engineer.md) related ones.
+
+### Expectations
```

Comment:
Can we push it by one level? E.g.

```
## Expectations
### Process management
```

Thank you

Reply:
Moved one level up and repushed changes after rebase